tests/components/test_rflink.py

```diff
@@ -285,7 +285,7 @@ def test_error_when_not_connected(hass, monkeypatch):
     failures = [False, True, False]
 
     # setup mocking rflink module
-    _, mock_create, _, disconnect_callback = yield from mock_rflink(
+    _, _, _, disconnect_callback = yield from mock_rflink(
```

Comment:
We could clean up this module too to use await syntax in the future.

Reply:
@MartinHjelmare Just created PR #19551 but I have encountered some problems. If you were kind enough to take a look, I would be very grateful.

Thanks in advance.